### ISSUE TITLE
flash zip fails with : Busybox setup failed

### ISSUE THREAD

**Issue by SynapseScribe** [SmartPack/SmartFlasher#12]
trying to flash ex (elemenatlalX) kernel zip, but after inflating zip, flashing fails with error:

update-binary
inflating: tools/magiskboot
inflating: tools/busybox
inflating: tools/magiskpolicy
Busybox setup failed. Aborting...

The detailed log shows: "Mounting root filesystem: Failed * Please Note: Flashing may not work properly on this device!"

However I am rooted with magisk and have allowed the app root access. Also I have "Busybox for Android NDK" magisk module installed.

**Comment by sunilpaulmathew** [comment]
It's a totally different issue (not related to root/busybox access for the app) that I'm struggling to deal with for a while.  It sadly affects some modern devices. I guess you're in Android 10/11. From the technical side, the app will create a recovery-like environment and send the flashing command. The rest of the errors are from the anykernal installer (not the app).  The issue is most likely related to the failed mounting /system partition. If you have time and interest, try manually mounting the /system partition (mount -o, remound,rw /system) and try to continue flashing. In case if you got the same mounting issue, try to find the exact path to system partition from /proc/mounts.

**Comment by SynapseScribe** [comment]
Yes, i am on android 10. However afaik you cannot manually remount /system in android 10 as is locked as read only.
This was described here:
https://mobile.twitter.com/topjohnwu/status/1170404631865778177
https://forum.xda-developers.com/t/mount-system-as-rw-on-android-10-with-magisk-20-4.4081043/page-3

**Comment by devsk** [comment]
Any updates on this issue? Kernel flashing is broken on Android 11....:-(
